draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -340,23 +340,27 @@
 	  using the AES-CCM-16-64-128 algorithm.
 	</t>
 	<t>
-	  The following example CWT Claims Set of a CWT (using JSON notation) illustrates
+	  The following example CWT Claims Set of a CWT (using CBOR diagnostic notation) illustrates
 	  the use of an encrypted symmetric key as the
 	  <spanx style="verb">Encrypted_COSE_Key</spanx> member value:
 	</t>
 	<figure>
 	  <artwork><![CDATA[
   {
-   "iss": "https://server.example.com",
-   "sub": "24400320",
-   "aud": "s6BhdRkqt3",
-   "exp": 1311281970,
-   "iat": 1311280970,
-   "cnf":{
-     "Encrypted_COSE_Key":
-       "(TBD)"
-     }
+   /iss/ 1 : "coaps://server.example.com",
+   /sub/ 2 : "24400320",
+   /aud/ 3: "s6BhdRkqt3",
+   /exp/ 4 : 1311281970,
+   /iat/ 5 : 1311280970,
+   /cnf/ 8 : { /COSE_Encrypt0/
+    [
+     /protected header / h'A1010A' /{ \alg\ 1:10 \AES-CCM-16-64-128\}/,
+     /unprotected header/ { / iv / 5: h'13EACB7BC05DBC872E8010C0E3'},
+     /ciphertext/h'C4E52D9C1DBB52D2CE291C65FB64F3CD5A8811FE058A3AA4972F6DD600'
```

Comment:
Can you also add somewhere what the plaintext corresponding to the ciphertext contains and what the key was that was used to produce the ciphertext?